go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/topolvm/pvc-autoresizer
 
-go 1.18
+go 1.19
```

Comment:
It seems to build staticcheck, however, we want to save the version because pvc-autoresizer dependencies (especially kubernetes related libraries) are compatible with 1.18.
Could you fix the version of staticcheck instead?

Reply:
@bells17 
I apologize for wasting your time. We now use Go 1.19 (https://github.com/topolvm/pvc-autoresizer/pull/177), so rebase on the main branch and erase the commit (https://github.com/topolvm/pvc-autoresizer/pull/169/commits/511fc5fe1e3635401e2aa5eefc458f617ece6cb2).